helpers/wultrunner/Makefile

```diff
@@ -13,7 +13,7 @@ LIBBPF ?=
 # Paths to include when compiling eBPF code.
 BPF_INC := -I$(KSRC)/include \
 	   -I$(KSRC)/usr/include \
-	   -I$(KSRC)/libbpf/include \
+	   -I$(KSRC)/tools/bpf/resolve_btfids/libbpf \
```

Comment:
The path you add is Fedora-specific. It does not exist in non-Fedora cases. So this patch causes failures with non-Fedora kernels.

Instead, could you define 'BPF_INC' in the build script/spec file, and pass it to make?